jetty-server/src/main/java/org/eclipse/jetty/server/session/DefaultSessionIdManager.java

```diff
@@ -268,7 +270,7 @@ public String newSessionId(long seedTerm)
                 if (!StringUtil.isBlank(_workerName))
                     id = _workerName + id;
 
-                id = id + COUNTER.getAndIncrement();
+                id = id + Long.toString(COUNTER.getAndIncrement());
```

Comment:
I think i've seen this go from Long.toString() to none to back again in multiple commits on this PR.

Reply:
Been asked to not do other changes that were not related to synchronization.